lib/index.js

```diff
@@ -236,15 +236,36 @@ if (!config.ignoreConflicts) {
 // if the service key is defined in the environment then use that. if
 // not see if it is defined in the config file.
 //
-let serviceKey = env.APPOPTICS_SERVICE_KEY || config.serviceKey
-exports.serviceKey = serviceKey
+const environmentKey = process.env.APPOPTICS_SERVICE_KEY
```

Comment:
Do you need `process.` when you defined `const env = process.env` above?

Reply:
I don't think so. The reason it's there is that I had a bug (I'd forgotten to change the parameter in the call to oboeInit()) and I added that on the wild chance that process itself used a proxy. I'm pretty sure I can change it back but testing this (and making sure it's not oboe changing the value) is a bit painful, so I just left it.

nice eye!